lib/support/changelog.js

```diff
@@ -30,17 +35,24 @@ var changelog = module.exports = {
   },
 
   needsUpdate: function () {
-    return this.read().indexOf('## Upcoming') > -1;
+    var changelog = this.read();
+
+    return this.needles.map(function (needle) {
+      return changelog.indexOf(needle) > -1;
+    }).indexOf(true) > -1;
```

Comment:
what about rewriting this using `some`?

```js
return this.needles.some(function (needle) {
     return changelog.indexOf(needle) > -1;
});
```

[some function on mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)